api-microservice/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:12-alpine as builder
```

Comment:
Bloke who gave you that is a clever chap

Reply:
He is! :-)